src/pamiq_core/torch/model.py

```diff
@@ -180,3 +181,21 @@ def sync_impl(self, inference_model: TorchInferenceModel[T]) -> None:
     def forward(self, *args: Any, **kwds: Any) -> Any:
         """forward."""
         return self.model(*args, **kwds)
+
+    @override
+    def save_state(self, path: Path) -> None:
+        """Save the model params.
+
+        Args:
+            path: Path where the states should be saved.
+        """
+        torch.save(self.model.state_dict(), path)  # pyright: ignore[reportUnknownMemberType]
```

Comment:
pathに拡張子は無い想定なのでつける処理もお願いします。pytorchの拡張子は `pt`だそうです。
```suggestion
        torch.save(self.model.state_dict(), f"{path}.pt")  # pyright: ignore[reportUnknownMemberType]
```

Reply:
https://github.com/MLShukai/pamiq-core/pull/147/commits/50b71e847f94afa6950ec8c40a091271fb4ac443 で修正